core-bundle/src/Resources/contao/library/Contao/File.php

```diff
@@ -349,10 +346,14 @@ public function __get($strKey)
 				return $this->arrImageViewSize;
 
 			case 'viewWidth':
-				return $this->imageViewSize !== false ? $this->imageViewSize[0] : null;
+				$imageViewSize = $this->imageViewSize;
```

Comment:
Why did you add a separate variable here?

Reply:
`empty($this->imageViewSize)` will always be true, because it accesses `__isset` first and not `__get` and `__isset` is not implemented in `Contao\File`.

Reply:
I see. We should definitely add a comment here. 😄